### DISCUSSION TITLE
"System-ui" font family in foliate.

### DISCUSSION THREAD

**Discussion by tuxnm** [johnfactotum/foliate#985] [Q&A] [answered]
Thanks for developing this nice ebook reader, i just want to know about what font is "System-UI" in a font selection list of the foliate menu.

**Comment by johnfactotum** [comment, marked as answer]
It probably depends on your system, but for me it's Cantarell. Strangely, it doesn't seem to be affected by fontconfig or GTK's font settings, so I'm not sure where it's coming from.

Also note that after applying the font, you can [use WebKit's inspector](https://github.com/johnfactotum/foliate/wiki#for-publishers-and-developers) to see which font is actually used.
